.buildkite/pipeline.yml

```diff
@@ -1,6 +1,8 @@
 env:
   SETUP_GVM_VERSION: 'v0.5.1' # https://github.com/andrewkroh/gvm/issues/44#issuecomment-1013231151
   ELASTIC_PACKAGE_COMPOSE_DISABLE_ANSI: "true"
+  # ELASTIC_PACKAGE_COMPOSE_DISABLE_PROGRESS_OUTPUT: "true"
+  ELASTIC_PACKAGE_COMPOSE_PROGRESS_OUTPUT: "quiet"
```

Comment:
@jsoriano any preference about which environment would be best to set ? 

ELASTIC_PACKAGE_COMPOSE_DISABLE_PROGRESS_OUTPUT would be just to set `true` or `false`, and we would need to decide if we want `plain` or `quiet` for the `--progress` flag.

In ELASTIC_PACKAGE_COMPOSE_PROGRESS_OUTPUT var, a few values would be allowed (auto, plain or quiet) and the user could choose which one.